src/ui/server/controllers/insurance/no-access-application-submitted/index.test.ts

```diff
@@ -0,0 +1,34 @@
+import { TEMPLATE, get } from '.';
+import { PAGES } from '../../../content-strings';
+import { TEMPLATES } from '../../../constants';
+import insuranceCorePageVariables from '../../../helpers/page-variables/core/insurance';
+import { Request, Response } from '../../../../types';
+import { mockReq, mockRes } from '../../../test-mocks';
+
+describe('controllers/insurance/no-access-application-submitted', () => {
+  let req: Request;
+  let res: Response;
+
+  beforeEach(() => {
+    req = mockReq();
+    res = mockRes();
+  });
+
+  describe('TEMPLATE', () => {
+    it('should have the correct template defined', () => {
+      expect(TEMPLATE).toEqual(TEMPLATES.INSURANCE.NO_ACCESS_APPLICATION_SUBMITTED);
+    });
+  });
+
+  describe('get', () => {
+    it('should render template', () => {
+      get(req, res);
+
+      const expectedVariables = {
```

Comment:
Don't need the spread operator here can just be `expectedVariables = insuranceCorePageVariables({ ...`